code/game/gamemodes/revolution/revolution.dm

```diff
@@ -393,11 +220,10 @@
 	if(foecount == scoreboard.score_arrested)
 		scoreboard.all_arrested = TRUE
 
-	for(var/I in GLOB.human_list)
-		var/mob/living/carbon/human/H = I
-		if(H.stat == DEAD && H.mind?.assigned_role)
-			if(H.mind.assigned_role in list("Captain", "Head of Security", "Head of Personnel", "Chief Engineer", "Research Director"))
-				scoreboard.score_dead_command++
+	for(var/datum/mind/head_mind in get_all_heads())
+		var/mob/living/carbon/human/H = head_mind.current
+		if(H.stat == DEAD)
```

Comment:
```suggestion
		if(!isnull(H) && H.stat == DEAD)
```

Reply:
I'm going to change this to `isnull(H) || H.stat == DEAD` because if they have no body, and are a target, they're probably dead.